web-js/README.md

```diff
@@ -0,0 +1,54 @@
+# MongoDB Realm Web Starter
+
+This project contains the web client for the MongoDB Realm template application. The client allows you to:
+- Sign up and log in as an Email/Password user
+- Create, check off, and delete to-do items
+
+The client allows you to connect over Realm's [GraphQL API](https://docs.mongodb.com/realm/graphql/) or using [remote MongoDB queries](https://docs.mongodb.com/realm/web/mongodb/).
+
+> ⚛️ **Create React App**
+> This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## How To Build & Run the App
+
+### 1. Download & Install Dependencies
+
+You can download the app from the Realm UI, with Realm CLI, or by cloning this repository:
+
+```shell
+gh repo clone mongodb-university/realm-template-apps
+cd realm-template-apps/web-js
+npm install
+```
+
+### 2. Define the API Type
+
+This app is built to use a pluggable backend API that conforms to the useTodo() hook interface. You
+can either use a pure MQL implementation or use your app's GraphQL API.
+
+Open the `.env` file and specify either `"graphql"` or `"mql"` for `REACT_APP_API_TYPE`:
+
+```shell
+REACT_APP_API_TYPE="graphql"
+# REACT_APP_API_TYPE="mql"
+```
+
+> 💡 **Restart to Change Environment Values**
+> The app only sources `.env` values once. If you change a value while the app is running, the app continues to use the value that was defined when it started.
+
+### 3. Specify Your Realm App ID
```

Comment:
I originally had them there actually, but the [spec](https://docs.google.com/document/d/18vFW-ATTH7W2S-8BM3vZj5WGqWc66jPV_UoR25bPW6Y/edit) has JS using a `realm.json` file (iOS uses `Realm.plist`) so that the backend server can inject the user's App ID & URL in a platform-agnostic way.

Reply:
Oh okay 👍  makes sense then.